### PR TITLE
TiCDC: modify Kafka sink uri format

### PR DESCRIPTION
**Pull request by wk989898**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)
TiCDC supports multiple Kafka addresses, but the document does not indicate this. And `userinfo@` is unused.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [x] v8.2 (TiDB 8.2 versions)
- [ ] v8.1 (TiDB 8.1 versions)
- [ ] v8.0 (TiDB 8.0 versions)
- [ ] v7.6 (TiDB 7.6 versions)
- [ ] v7.5 (TiDB 7.5 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s): https://asktug.com/t/topic/1026783
### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: ticdc/ticdc-sink-to-kafka.md [hunk 1]
```diff
@@ -36,13 +36,13 @@ Info: {"sink-uri":"kafka://127.0.0.1:9092/topic-name?protocol=canal-json&kafka-v
 Sink URI is used to specify the connection information of the TiCDC target system. The format is as follows:
 
 ```shell
-[scheme]://[userinfo@][host]:[port][/path]?[query_parameters]
+[scheme]://[host]:[port],[host]:[port],[host]:[port][/path]?[query_parameters]
```
Comment (qiancai) [inline ticdc/ticdc-sink-to-kafka.md:39]:
```suggestion
[scheme]://[host]:[port][/path]?[query_parameters]
```

File: ticdc/ticdc-sink-to-kafka.md [hunk 2]
```diff
@@ -36,13 +36,13 @@ Info: {"sink-uri":"kafka://127.0.0.1:9092/topic-name?protocol=canal-json&kafka-v
...
 ```
```
Comment (qiancai) [inline ticdc/ticdc-sink-to-kafka.md:41]:
````suggestion
If there are multiple hosts or ports for the downstream Kafka, you can add multiple hosts or ports in sink URI too. For example:

```shell
[scheme]://[host]:[port],[host]:[port],[host]:[port][/path]?[query_parameters]
```

````

File: ticdc/ticdc-sink-to-kafka.md [hunk 3]
```diff
@@ -36,13 +36,13 @@ Info: {"sink-uri":"kafka://127.0.0.1:9092/topic-name?protocol=canal-json&kafka-v
...
 
 Sample configuration:
 
 ```shell
---sink-uri="kafka://127.0.0.1:9092/topic-name?protocol=canal-json&kafka-version=2.4.0&partition-num=6&max-message-bytes=67108864&replication-factor=1"
+--sink-uri="kafka://127.0.0.1:9092,127.0.0.2:9093,127.0.0.3:9094/topic-name?protocol=canal-json&kafka-version=2.4.0&partition-num=6&max-message-bytes=67108864&replication-factor=1"
 ```
 
 The following are descriptions of sink URI parameters and values that can be configured for Kafka:
```
Comment (qiancai) [inline ticdc/ticdc-sink-to-kafka.md:48]: 下面表格里的前两行，或许也需要对应修改下

image

比如改成
```
| 127.0.0.1:9092,127.0.0.2:9093,127.0.0.3:9094          | A list of IP addresses and ports of the downstream Kafka services.                                 |
```

File: ticdc/ticdc-sink-to-kafka.md (hunk 1 above)
Comment (wk989898) [reply ticdc/ticdc-sink-to-kafka.md:39]: we need to indicate `Sink URL` supports multiple hosts.

File: ticdc/ticdc-sink-to-kafka.md [hunk 4]
```diff
@@ -36,13 +36,18 @@ Info: {"sink-uri":"kafka://127.0.0.1:9092/topic-name?protocol=canal-json&kafka-v
 Sink URI is used to specify the connection information of the TiCDC target system. The format is as follows:
 
 ```shell
-[scheme]://[userinfo@][host]:[port][/path]?[query_parameters]
+[scheme]://[host]:[port][/path]?[query_parameters]
+```
+If there are multiple hosts or ports for the downstream Kafka, you can add multiple hosts or ports in sink URI too. For example:
```
Comment (qiancai) [inline ticdc/ticdc-sink-to-kafka.md:41]:
```suggestion

If there are multiple hosts or ports for the downstream Kafka, you can add multiple `[host]:[port]` in sink URI too. For example:
```

File: ticdc/ticdc-sink-to-kafka.md [hunk 5]
```diff
@@ -36,9 +36,17 @@ Info: {"sink-uri":"kafka://127.0.0.1:9092/topic-name?protocol=canal-json&kafka-v
 Sink URI is used to specify the connection information of the TiCDC target system. The format is as follows:
 
 ```shell
-[scheme]://[userinfo@][host]:[port][/path]?[query_parameters]
+[scheme]://[host]:[port][/path]?[query_parameters]
 ```
 
+> **Tip:**
+> 
+> If there are multiple hosts or ports for the downstream Kafka, you can add multiple `[host]:[port]` in the sink URI. For example:
```
Comment (qiancai) [inline ticdc/ticdc-sink-to-kafka.md:44]:
```suggestion
> If there are multiple hosts or ports for the downstream Kafka, you can configure multiple `[host]:[port]` in the sink URI. For example:
```

**Review by Oreoxmt — APPROVED** [review]

**Pull request merged**